models/relations.py

```diff
@@ -46,6 +46,15 @@ def valid_construction(cls, user_id, first, last, death_year, is_deceased, gende
 
     
     def json(self):
+        return {
+            "id": self.id,
```

Comment:
what's the difference between `json()` and `individual_json()`

Reply:
json() is for returning list of relations for a user, while individual_json() is for returning all the parameters of a single relation. Is there a way to use the same function to return different parameters?

Reply:
I can't immediately see how `json()` would return a list. You do realize these functions share the same lines of code? Both `json()` and `individual_json()` return a dictionary.